orchagent/aclorch.cpp

```diff
@@ -1924,6 +1936,16 @@ bool AclRuleMirror::activate()
         setAction(it.first, attr.value.aclaction);
     }
 
+    // If the rule with mirror action is removed and then mirror is activated, create the counter before rule is created
+    if (!hasCounter())
```

Comment:
@bingwang-ms to check this change as these are generic changes, not specific to VOQ. wondering why this is not affecting non-chassis platforms.

Reply:
Normally, we will not see this issue because the counter for the rule has been created at `AclRule::create()`. It can happen if the rule is deleted before the mirror session is activated. It's a corner case, I think.

Reply:
This happens when the acl rule is created with MIRROR action and  deleted before the nexthop to the mirror destination is not resolved and activated. The counter is created only when the mirror session is activated.